usr/sbin/pkcsep11_migrate/pkcsep11_migrate.c

```diff
@@ -100,13 +97,11 @@ static int reencrypt(CK_SESSION_HANDLE session, CK_ULONG obj, CK_BYTE *old)
     target_t target = XCP_TGT_INIT;
     CK_RV rc;
     CK_BYTE name[256];
-
-    ep11_opaque *op_old = (ep11_opaque *) old;
-    ep11_opaque op_new;
-
+    unsigned char blob[blobsize];
```

Comment:
Please rework the code and rename blobsize to BLOBSIZE.

Reply:
Fixed